.gitignore

```diff
@@ -40,3 +40,4 @@ dataactcore/scripts/dropMany.py
 runDynamo.py
 dataactvalidator/config/sf_133*.csv
 runBothTestPackages.py
+dataactcore/dev_config.yml
```

Comment:
This was added to the gitignore while I experimented with an env specific config, given the changes above I'll change this to local_config and use that instead